### DISCUSSION TITLE
github and docker hub version difference

### DISCUSSION THREAD

**Discussion by waimingsky** [ldproxy/ldproxy#740] [Setup] [answered]
Hi there, 
running "docker run --name ldproxy -d -p 7080:7080 -v ~/ldproxy_data:/ldproxy/data iide/ldproxy:latest" after "docker pull iide/ldproxy" from docker hub, the attribute under features can be extracted from esri dataset; however, while running "./gradlew run", which is clone form "https://github.com/interactive-instruments/ldproxy.git" on github, the attrubute cannot be read. any thought? is there any difference between the two versions? thanks

**Comment by azahnen** [comment, marked as answer]
Hi, 
`iide/ldproxy:latest` is actually the latest stable release, so it corresponds to [v3.2.4](https://github.com/interactive-instruments/ldproxy/releases/tag/v3.2.4) on Github. 
The master branch, which I guess you checked out, corresponds to `iide/ldproxy:next`.
